apps/admin-gui-e2e/src/integration/vo-management-perun_admin.spec.js

```diff
@@ -12,26 +12,268 @@ context('Actions', () => {
   })
 
   beforeEach(() => {
-    cy.visit('home')
-    cy.intercept('**/vosManager/createVo/**').as('createVo')
+    cy.visit('organizations');
   })
 
   it('test create vo', () => {
 
-    cy.get('#access-item-btn')
-        .click()
-      .get('#new-vo-btn')
+    cy.intercept('**/vosManager/createVo/**').as('createVo')
+      .get('[data-cy=new-vo-button]')
         .click()
-      .get('#vo-name-input')
+      .get('[data-cy=vo-name-input]')
         .type('test-e2e-vo')
-      .get('#vo-shortname-input')
+      .get('[data-cy=vo-shortname-input]')
         .type('test-e2e-vo')
-      .get('#create-vo-button')
+      .get('[data-cy=create-vo-button]')
         .click()
       .wait('@createVo')
       // assert that the vo was created
-      .get('#vo-name-link')
+      .get('[data-cy=vo-name-link]')
         .contains('test-e2e-vo')
         .should('exist')
-  })
+  });
+
+  it('test add attribute and delete', () => {
+
+    cy.intercept('**/attributesManager/setAttributes/vo').as('setAttributes')
+      .intercept('**/attributesManager/removeAttributes/**').as('removeAttributes')
+      .get('[data-cy=test-e2e-vo-from-db]')
```

Comment:
I would suggest creating constants with names of objects from db. For example:
```Typescript
const dbVoName = 'test-e2e-vo-from-db';
```
Then you could use it here like this: 
```suggestion
      .get(`[data-cy=${dbVoName}]`)
```
These values should always be defined as contacts. It is then much more clear, where the values are. Also, if you use it in multiple places, you are guaranteed to use the same value.